maven/bnd-export-maven-plugin/src/main/java/aQute/bnd/maven/export/plugin/ExportMojo.java

```diff
@@ -100,9 +110,16 @@
 
 	@Override
 	public void execute() throws MojoExecutionException, MojoFailureException {
+		if (skip) {
+			logger.debug("skip project as configured");
+			return;
+		}
+
 		try {
 			DependencyResolver dependencyResolver = new DependencyResolver(project, repositorySession, resolver,
-				system);
+				system, scopes.stream()
+					.map(Scope::name)
+					.collect(Collectors.toList()));
```

Comment:
Perhaps this logic should be centralized in DependencyResolver with a constructor that takes a `Set<Scope>` argument?

Reply:
sure!